### DISCUSSION TITLE
Is it possible to not show the close tab button on the tab bar?

### DISCUSSION THREAD

**Discussion by gsuuon** [wezterm/wezterm#3615] [Q&A]
Hello! Is there a config to remove this close tab button in fancy tab bar mode? I've scoured the docs but wasn't able to find it.
![image](https://user-images.githubusercontent.com/6422188/234191578-c0963736-8c06-4a05-b3af-a7a796454f31.png)

**Comment by yutkat** [comment]
If you want to customize tab title, you can turn off `fancy_tab_bar` and set it yourself using `format-tab-title`.

```lua
use_fancy_tab_bar = false
```

https://wezfurlong.org/wezterm/config/lua/window-events/format-tab-title.html

my tab-title is here.
![swappy-20230425-221955](https://user-images.githubusercontent.com/8683947/234289256-ecac9f34-1ca2-4397-8171-4f5618849ed2.png)
https://github.com/yutkat/dotfiles/blob/11dba6baf4d382c30bc37702032396ac904205ea/.config/wezterm/on.lua#L35-L71

**Comment by gsuuon** [comment]
I tried that but I wasn't able to change the font size separately for the tab bar. Did I do it wrong or is that not possible?

**Reply by yutkat** [reply]
Currently I think if you don't use `fancy_tab_bar` (use Retro style), you cannot change only the font size of the tabs. You have to change the entire font size.

If you want to do that, you can wait for this Issue to be resolved.
https://github.com/wez/wezterm/issues/1180

https://wezfurlong.org/wezterm/config/appearance.html?h=fancy#retro-tab-bar-appearance

**Reply by yutkat** [reply]
@gsuuon Have you solved this?

**Reply by gsuuon** [reply]
@yutkat no I've just learned to accept it 🙂. It'd be nice to get rid of it since I never use the mouse to click up there and would prefer more space for the tab titles. For now I'd rather keep the smaller font size in tabs and ignore the x's than have to use the same font size throughout since sometimes I go huge with my terminal font size [like this.](https://github.com/wez/wezterm/discussions/628#discussioncomment-5735102)

**Reply by yutkat** [reply]
Understood. It would be better to wait for this issue to be resolved.
https://github.com/wez/wezterm/issues/1180
